packages/db/package.json

```diff
@@ -29,10 +29,6 @@
     "clean": "rm -rf ./dist ./types/schema.d.ts",
     "madge": "madge ./src/index.ts",
     "prepare": "yarn build",
-    "start": "ts-node-dev -r tsconfig-paths/register ./bin/server",
-    "start:debug": "DEBUG=pouchdb:api ts-node-dev -r tsconfig-paths/register ./bin/server",
-    "start:drizzle": "ts-node-dev -r tsconfig-paths/register ./bin/server ./test/truffle-projects/drizzle-box",
-    "start:drizzle:debug": "DEBUG=pouchdb:api ts-node-dev -r tsconfig-paths/register ./bin/server ./test/truffle-projects/drizzle-box",
```

Comment:
We can lose the `truffle-projects/` folder now.

Reply:
Oh, I didn't realize, thanks!  @gnidan didn't mention that so I didn't know offhand whether that was used elsewhere.  (I guess I should likely check myself to be sure...)

Reply:
Yup, looks like you're right!  Made that change.